beeline.go

```diff
@@ -10,16 +10,18 @@ import (
 	"github.com/honeycombio/libhoney-go/transmission"
 
 	"github.com/honeycombio/beeline-go/client"
+	"github.com/honeycombio/beeline-go/propagation"
 	"github.com/honeycombio/beeline-go/sample"
 	"github.com/honeycombio/beeline-go/trace"
 	libhoney "github.com/honeycombio/libhoney-go"
 )
 
 const (
-	defaultWriteKey   = "apikey-placeholder"
-	defaultDataset    = "beeline-go"
-	defaultSampleRate = 1
-	warningColor      = "\033[1;33m%s\033[0m"
+	defaultWriteKey    = "apikey-placeholder"
+	defaultDataset     = "beeline-go"
+	defaultServiceName = "unknown_service:go"
```

Comment:
Can we detect process name in the fallback? https://github.com/open-telemetry/opentelemetry-go/blob/c4ba4811a2488128fbe1d6a98c06d80157f4bef1/sdk/resource/builtin.go#L96